source/documentation/deploying_services/index.md

```diff
@@ -6,7 +6,11 @@ Many 12-factor applications rely on backing services such as a database, an emai
 
 In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command.
 
-Currently, the only services available in the marketplace are the backing [PostgreSQL database service](/#using-postgresql) and [MySQL database service](/#using-mysql).
+Currently, the following database services are available in the marketplace:
+[PostgreSQL](/#using-postgresql)
+[MySQL](/#using-mysql)
+MongoDB
```

Comment:
MongoDB isn't available generally in the marketplace yet, only to some people who are giving us feedback on what's available so far.